shard/src/ext/pug_cog.py

```diff
@@ -0,0 +1,60 @@
+from discord.ext import commands
+from contextlib import suppress
+import time
+import asyncio
+from lib.config import logger
+
+
+class Pug(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def startpug(self, ctx, requiredPlayers: int = 10):
```

Comment:
I kinda think there should be some indicator of the game, and I also think allowing negatives is scuffed

`async def startpug(self, ctx, requiredPlayers: lambda p: abs(int(p)) = 10, game: str = ""):`

Reply:
hmm, probably want to do validation to make sure someone doesn't put 0 either. The game string is an interesting add and I have an idea on how to implement it.